cebong/templates/base.html

```diff
@@ -76,7 +76,7 @@
 
         <footer id="contentinfo" class="body">
                 <address id="about" class="vcard body">
-                    {{FOOTERTEXT}} Proudly powered by <a href="http://getpelican.com/">Pelican</a>, which takes great advantage of <a href="http://python.org">Python</a>. Theme is using <a href="http://github.com/getpelican/pelican-themes" target="_blank">cebong</a>.
+                    {{FOOTERTEXT}} Proudly powered by <a href="http://getpelican.com/">Pelican</a>, which takes great advantage of <a href="https://www.python.org/">Python</a>. Theme is using <a href="http://github.com/getpelican/pelican-themes" target="_blank">cebong</a>.
```

Comment:
The GitHub link on this line can be HTTPS too:

```
http://github.com/getpelican/pelican-themes
```

Would you like to change that one too?